tests/acceptance/book/CreatePartCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use \Codeception\Util\HttpCode;
+use Page\Acceptance\CreateBook;
+use Page\Acceptance\CreateBookPart;
+
+class CreatePartCest
+{
+	public $bookURL = 'testbookpart';
+
+	public function _before(AcceptanceTester $I, CreateBook $createBookPage)
+	{
+		$I->loginAsAdmin();
+
+		$createBookPage->createBook( $this->bookURL, 'Test Book Part', true );
+	}
+
+	public function tryToCreateBookPart(AcceptanceTester $I, CreateBookPart $createBookPartPage)
+	{
+		$I->amOnPage( "$this->bookURL/wp-admin/admin.php?page=pb_organize" );
+		$I->dontSee( $partName = 'Part '.mt_rand() );
```

Comment:
The test looks good. I'm curious why the variable $partName is created inside the function.

Reply:
This is just to reduce the lines of code. I can change that to be more explicit.